std/portable/index.js

```diff
@@ -244,10 +244,19 @@ globalScope["fmodf"] = function fmodf(x, y) {
 };
 
 globalScope["JSMath"] = Math;
```

Comment:
Looks like this would benefit from a `var JSMath = globalScope["JSMath"] = Math;` and using JSMath below to reduce verbosity. Wdyt?

Reply:
Yeah, makes sense

Reply:
I decide use `Object.defineProperties`. This best practice for making such polyfills

Reply:
Makes sense, yeah :)